plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -320,6 +319,13 @@ void GSSettingsDlg::UpdateRenderers()
 	{
 		GSRendererType ini_renderer = GSRendererType(theApp.GetConfigI("Renderer"));
 		renderer_setting = (ini_renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : ini_renderer;
+
+		if (ini_renderer == GSRendererType::Undefined) {
+			if (renderer_setting == GSRendererType::OGL_HW)
+				SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_SETCURSEL, 2, 0);
+			else if (renderer_setting == GSRendererType::DX9_HW || renderer_setting == GSRendererType::DX1011_HW)
```

Comment:
Couldn't this just be ``else`` instead ? CRC hack level is only visible for the three hardware renderers (OpenGL, D3D9 , D3D11) .